gradle/libs.versions.toml

```diff
@@ -63,6 +72,8 @@ asm = "org.ow2.asm:asm:9.7"
 
 [bundles]
 
+semconv = ["opentelemetrySemconv", "opentelemetrySemconvIncubating"]
```

Comment:
[for reviewer] this allows to get both the incubating/stable semconv at once, since they are both expected to be used together most of the time.